src/components/AnchorButton.js

```diff
@@ -0,0 +1,24 @@
+import styled, { css } from 'styled-components'
+
+const HoverSyles = css`
```

Comment:
Aren't we being repetitive with `HoverStyles` in various files, could we refactor to just reuse that at each instance.